### PR TITLE
Browser updates

### PR DESCRIPTION
**Pull request by BobGerman**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                                |
| New feature?    | yes                               |
| New sample?     | no                               |
| Related issues? |  |

## What's in this Pull Request?

The following code changes to the sample browser:
 - Now reads from local samples.json unless host name is pnp.github.io (production)
 - Now handles multiple query terms (implied "AND"); handles double quotes to allow spaces in a query term
 - Updated the display of the word "Features" to "Capabilities" per recent doc updates by Teams engineering

### CODE REVIEW

**Comment by hugoabernier** [comment]
@BobGerman that's a great update!!! I already implemented it on the webparts, extensions, and power platform sample repos.

File: docs/javascript/loadsamples.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 /**
  * This file is unique for each sample browser. It contains the logic specific to each repo for loading the samples as needed.
  */
-var jsonPath = "https://raw.githubusercontent.com/pnp/teams-dev-samples/gh-pages/samples.json";
+var jsonPath = "https://pnp.github.io/teams-dev-samples/samples.json";
+if (window.location.host.toLowerCase() !== "pnp.github.io") {
```
Comment (hugoabernier) [inline docs/javascript/loadsamples.js:5]: Technically, we could always load from the window.location.host URL (without the `if` statement), but this works great!

**Review by hugoabernier — COMMENTED** [review]
Works great! I implemented it in my own repos!

File: docs/javascript/loadsamples.js (hunk 1 above)
Comment (BobGerman) [reply docs/javascript/loadsamples.js:5]: Thanks; I think we discussed that one offline and it is resolved; thanks!

**Pull request merged**